### DISCUSSION TITLE
Whitelisting Meteostat IP

### DISCUSSION THREAD

**Discussion by bktllr** [meteostat/meteostat-python#124] [Q&A]
Hi,

I want to use the Meteostat Python Library in a closed-down Jupyter Lab environment. To make it work, I need to whitelist the IP address that is used to query and return the results. Would it be possible to provide me with the IP addresses that I need to whitelist to make the python library work?

Thanks in advance,

Jonas.

**Comment by clampr** [comment]
Hi Jonas,

You'll need to whitelist the IP addresses for https://bulk.meteostat.net/. This subdomain is using the Cloudflare CDN. The IP ranges used by Cloudflare are documented here: https://www.cloudflare.com/en-gb/ips/

**Reply by bktllr** [reply]
Great, thank you very much!

**Reply by javierhuertay** [reply]
Looking for the same thing thanks!
Maybe this can be posted somewhere else, like in the repository or in the error itself
